### PR TITLE
Add chinese translate

### PR DESCRIPTION
**Pull request by stigersmile**

### CODE REVIEW

**Review by t7yang — CHANGES_REQUESTED** [review]
你的進度已經落後目前的版本，請重新 pull 確保完成你本機的進度再開始翻譯。

除了提及的問題外，也請參考新上線的[正體中文翻譯指南](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md)並重新檢視你的翻譯內容確保符合中文翻譯的指導原則，感謝。

審核小組在經過討論後做了一個重大的決定——取消中英文之間的空格修飾，原因跟細節可以參考[這裡](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E4%B8%AD%E6%96%87%E5%8F%8A%E6%8B%89%E4%B8%81%E8%AA%9E%E7%B3%BB%E6%96%87%E5%AD%97%E9%96%93%E4%B8%8D%E5%8A%A0%E4%BF%AE%E9%A3%BE%E7%A9%BA%E7%99%BD)，再麻煩根據指導原則進行修改，造成困擾的部分請包涵。

**Comment by stigersmile** [comment]
fixed the conflict

File: files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/angular_item_component/index.html [hunk 1]
```diff
@@ -223,28 +212,17 @@ <h2 id="use_the_itemcomponent_in_the_appcomponent">Use the ItemComponent in the
   &lt;/li&gt;
 &lt;/ul&gt;</pre>
 
-<p>The double curly brace syntax, <code>\{{}}</code>, in the <code>&lt;h2&gt;</code> interpolates the length of the <code>items</code> array and displays the number.</p>
+<p>雙括號 <code>{{}}</code>，在 <code>&lt;h2&gt;</code> 內顯示 <code>items</code> 的長度與數目。</p>
```
Comment (tso1158687) [inline files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/angular_item_component/index.html:215]: 這一行修正試試看

**Comment by irvin** [comment]
現在與 main 分支還是有 conflict 喔

**Comment by stigersmile** [comment]
> 現在與 main 分支還是有 conflict 喔

請問在哪裡呢? 從我這邊看不出來

**Comment by irvin** [comment]
> > 現在與 main 分支還是有 conflict 喔
> 
> 請問在哪裡呢? 從我這邊看不出來

請在你的 repo 下按 fetch upstream 試試

截圖 2021-06-05 下午5 49 53

**Comment by SphinxKnight** [comment]
This pull request contains files unrelated to the PR itself. Including files for various locales (including frozen ones where contribution is not accepted). It should not be merged as is.

Let us know if you need help with Git to get this one on track :)

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Review comment: unrelated files should not be touched by this PR.

**Comment by stigersmile** [comment]
> 
> 
> This pull request contains files unrelated to the PR itself. Including files for various locales (including frozen ones where contribution is not accepted). It should not be merged as is.
> 
> Let us know if you need help with Git to get this one on track :)

Than you for your feedback, I just push "Fetch upstream" Fetch and merge on my github. What should I do to fixed this problem?

**Comment by t7yang** [comment]
I think we should stop here.

@stigersmile could you please submit a new PR and make sure your work is based on the latest main branch. This is easier than fixing the issue here if you unfamiliar with git.

Thx

**Pull request closed without merging**